src/@next/Icon/icons/ri-file-list-3.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 18 17" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.8333 16.8333H3.16663C2.50358 16.8333 1.8677 16.5699 1.39886 16.1011C0.930018 15.6323 0.666626 14.9964 0.666626 14.3333V0.999999C0.666626 0.778986 0.754423 0.567024 0.910704 0.410744C1.06698 0.254463 1.27895 0.166666 1.49996 0.166666H13.1666C13.3876 0.166666 13.5996 0.254463 13.7559 0.410744C13.9122 0.567024 14 0.778986 14 0.999999V11H17.3333V14.3333C17.3333 14.9964 17.0699 15.6323 16.6011 16.1011C16.1322 16.5699 15.4963 16.8333 14.8333 16.8333ZM14 12.6667V14.3333C14 14.5543 14.0878 14.7663 14.244 14.9226C14.4003 15.0789 14.6123 15.1667 14.8333 15.1667C15.0543 15.1667 15.2663 15.0789 15.4225 14.9226C15.5788 14.7663 15.6666 14.5543 15.6666 14.3333V12.6667H14ZM3.99996 4.33333V6H10.6666V4.33333H3.99996ZM3.99996 7.66667V9.33333H10.6666V7.66667H3.99996ZM3.99996 11V12.6667H8.16663V11H3.99996Z" fill="#666666"/>
```

Comment:
```suggestion
<path d="M14.8333 16.8333H3.16663C2.50358 16.8333 1.8677 16.5699 1.39886 16.1011C0.930018 15.6323 0.666626 14.9964 0.666626 14.3333V0.999999C0.666626 0.778986 0.754423 0.567024 0.910704 0.410744C1.06698 0.254463 1.27895 0.166666 1.49996 0.166666H13.1666C13.3876 0.166666 13.5996 0.254463 13.7559 0.410744C13.9122 0.567024 14 0.778986 14 0.999999V11H17.3333V14.3333C17.3333 14.9964 17.0699 15.6323 16.6011 16.1011C16.1322 16.5699 15.4963 16.8333 14.8333 16.8333ZM14 12.6667V14.3333C14 14.5543 14.0878 14.7663 14.244 14.9226C14.4003 15.0789 14.6123 15.1667 14.8333 15.1667C15.0543 15.1667 15.2663 15.0789 15.4225 14.9226C15.5788 14.7663 15.6666 14.5543 15.6666 14.3333V12.6667H14ZM3.99996 4.33333V6H10.6666V4.33333H3.99996ZM3.99996 7.66667V9.33333H10.6666V7.66667H3.99996ZM3.99996 11V12.6667H8.16663V11H3.99996Z" />
```
always make sure to remove the fill. one indicator is it should show in black on preview

Reply:
ah, okay. I was just look up to svg tag only, will take case of path tag as well